internal/auth/auth.go

```diff
@@ -152,6 +152,45 @@ func (s *AuthService) RemoveNamespaceFromUsers(ctx context.Context, namespace st
 	return nil
 }
 
+// AddGitconfigToUser will add the specified gitconfig to the User
+func (s *AuthService) AddGitconfigToUser(ctx context.Context, username, gitconfig string) error {
+	user, err := s.GetUserByUsername(ctx, username)
+	if err != nil {
+		return errors.Wrap(err, fmt.Sprintf("error getting user [%s] by username", username))
```

Comment:
Same for `Wrapf`

Reply:
Done with commit f6bc2eca5f02b7f0634afe443b9e2ea734faeb05